### DISCUSSION TITLE
Overwrite the current store with Horizon

### DISCUSSION THREAD

**Discussion by dexterdotbuzz** [swellstores/community#45] [Help] [answered]
Currently it doesn't overwrite the current Origin theme. Only allows for a new store. But how about the current users?

**Comment by dannyswell** [comment, marked as answer]
Hi @dexterdotbuzz!

<img width="750" alt="Screen Shot 2022-12-20 at 11 30 10 AM" src="https://user-images.githubusercontent.com/119523784/208718233-373613ab-2763-424b-9b3b-4ccac0b60d3f.png">

For customers, you're able to export them as a CSV or JSON file and reimport to the new store.
For users/admins in your settings section, currently the only way to do that is to re-invite them. 


As for overwriting the current theme, we recognize the use case and are planning to release a feature -- Storefront Dashboard v2 -- in the future, but as of now it's not possible.
